components/build/tekton-results/patch-postgres-statefulset.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: tekton-results-postgres
+  namespace: tekton-pipelines
+spec:
+  template:
+    spec:
+      volumes:
+        - name: run
+          emptyDir: {}
+        - name: data
+          emptyDir: {}
+      containers:
+        - name: postgres
+          image: quay.io/mkovarik/postgres:13
```

Comment:
Have you made any changes to official image or is it a way to pin/control it?

Reply:
No change, but wanted to avoid using dockerhub since there is pull limitation, when you have testing machine in the company network then it's usually failing. Wanted to switch to RH image but there is missing feature of applying sql files during initialization. Having this in TODO.

Reply:
> Wanted to switch to RH image but there is missing feature of applying sql files during initialization. Having this in TODO.

Is this a TODO to fix the postgresql image? :)

Reply:
I read it as follows
1. He is using an `unmodified container` copied from dockerhub to mkovarik to avoid the docker unauthenticated throttle. I assume this is from [ https://hub.docker.com/_/postgres ](https://hub.docker.com/_/postgres) and you are using using version 13 ? something like  .... 

```
docker pull postgres:13
docker tag docker.io/library/postgres:13 quay.io/mkovarik/postgres:13
docker push  quay.io/mkovarik/postgres:13
```

if we need to do this for a while, we should add it to the quay app-studio account so we can keep track and add to the build maintainers should we need to update

3. I see there is an an[ official v13 image from RH ](https://catalog.redhat.com/software/containers/rhel8/postgresql-13/5ffdbdef73a65398111b8362) which we could use but I assume from your comment that despite being same versions they are different in how they do initialization which means we cannot currently use it

Is this a bug or fearture of the RH v13 ?

Reply:
Fixed by creating additional configmap for execution of sql script.